### PR TITLE
Added validations to the automation tests for MySQL

### PR DESCRIPTION
**Pull request by priyanshi-yb**

### CODE REVIEW

File: .github/workflows/mysql-migtests.yml [hunk 1]
```diff
@@ -60,12 +60,12 @@ jobs:
     - name: "TEST: mysql-constraints"
       run: migtests/scripts/run-test.sh mysql-tests/mysql-constraints
 
-    - name: "TEST: mysql-case-sensitive"
-      run: migtests/scripts/run-test.sh mysql-tests/mysql-case-sensitive
-
```
Comment (amit-yb) [inline .github/workflows/mysql-migtests.yml:65]: Why?

File: migtests/lib/yb.py [hunk 2]
```diff
@@ -78,3 +79,40 @@ def get_objects_of_type(self, object_type, schema_name="public") -> List[str]:
...
+		cur = self.conn.cursor()
+		cur.execute(f"select sum({column_name}) from {schema_name}.{table_name}")
+		return cur.fetchone()[0]
+		
+	def get_count_index_on_table(self, schema_name="public") -> Dict[str,int]:
+		cur = self.conn.cursor()
+		cur.execute(f"SELECT tablename, count(indexname) FROM pg_indexes WHERE schemaname = '{schema_name}' GROUP  BY tablename;")
+		return {tablename: cnt for tablename,cnt in cur.fetchall()}
+
+	def get_distinct_values__of_column_of_table(self, table_name, column_name, schema_name="public") -> List[Any]:
```
Comment (amit-yb) [inline migtests/lib/yb.py:93]: Do you really intend to use double underscores in the function name?

File: yb-voyager/src/srcdb/common.go [hunk 3]
```diff
@@ -44,7 +44,7 @@ func processImportDirectives(fileName string) error {
 	// Create a new scanner and read the file line by line.
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
-		line := strings.Trim(scanner.Text(), " ")
+		line:=scanner.Text()
```
Comment (amit-yb) [inline yb-voyager/src/srcdb/common.go:47]: Create a separate PR for this issue. Generally, do not mix unrelated changes into same PR.

**Review by amit-yb — COMMENTED** [review]
LGTM.
Just publish a separate PR for the "single line function" issue. Then I will approve this PR.

File: yb-voyager/src/srcdb/common.go (hunk 3 above)
Comment (priyanshi-yb) [reply yb-voyager/src/srcdb/common.go:47]: Okay, raised.

File: migtests/lib/yb.py (hunk 2 above)
Comment (priyanshi-yb) [reply migtests/lib/yb.py:93]: Oh no, it must be by mistake.

File: .github/workflows/mysql-migtests.yml (hunk 1 above)
Comment (priyanshi-yb) [reply .github/workflows/mysql-migtests.yml:65]: As the case-sensitive case doesn't come for MySQL as MySQL table names are itself case-sensitive and and it doesn't allow quotes. This is not required and I confirmed it from @shubham-yb  as well.

**Review by amit-yb — APPROVED** [review]
